pkg/framework/coremodel/helpers.go

```diff
@@ -54,13 +55,19 @@ func doLoadFrameworkCUE(ctx *cue.Context) (v cue.Value, err error) {
 	}
 
 	over := make(map[string]load.Source)
-	err = tload.ToOverlay(prefix, m, over)
+
+	absolutePath, err := filepath.Abs(prefix)
+	if err != nil {
+		return
```

Comment:
i _think_ this actually returns the wrong `err`, because your `:=` assignment shadowed the signature-named return value.